### DISCUSSION TITLE
amazon-wishlist-alternatives/

### DISCUSSION THREAD

**Discussion by giscus** [JakeSteam/blog-personal#7] [Comments]
# amazon-wishlist-alternatives/

For many years I’ve had a public Amazon wishlist that I can share with friends & family. I add to it throughout the year, and it lets people buy me things I’ll definitely love. This has worked well, but recently Amazon removed support for third party products… so here’s where to migrate to.

https://jakelee.co.uk/amazon-wishlist-alternatives/

**Comment by AlexMoonsift** [comment]
Hi Jake, thanks for this article and really interesting to hear you are using a Github repo. I've not heard anyone using that for a wishlist before.

Have you tried Moonsift.com? It's a free wishlist that you can add to directly from any store using a Chrome extension or iOS app. Free to use, no ads, clean design and you can share wishlists with a single link.

I'm part of the team working on it so would be really interested in your feedback if you get a chance to try it out.

**Reply by bwims** [reply]
Not impressed!  Every time I clicked on the chrome extension, it said I was not logged in and invited me to do so.  I clicked the button, the web site agreed I was logged in, but the extension could not see it.  I use SRware Iron as my browser which works fine with all other extensions.

**Reply by bwims** [reply]
PS same problem with Edge.

**Reply by AlexMoonsift** [reply]
We've never heard of anyone using it on a SRware Iron browser before so thank you for that feedback (to be honest I'd never heard of SRware Iron before at all).

Chrome are making all extensions move to Manifest V3 (so we updated the extensions for those new guidelines) perhaps this is part of why it doesn't work on non-Chrome browsers at the moment.

People on recent versions of Chrome have no issue, but we notice some people on very very outdated versions of Chrome also run into some login problems.

Any screenshots or if you would be willing to send any errors you are getting, that would be extremely helpful.

Thanks for taking the time, we appreciate the feedback.

Alex

**Reply by AlexMoonsift** [reply]
We are also working on 'Add product from link' function, which will allow people to use it on any browser - including Safari, Firefox etc.

**Comment by bwims** [comment]
Ok, but it will have to wait for a couple of weeks as I will away on
holiday until then.

Regards

Brian

**Reply by AlexMoonsift** [reply]
No worries, that should give us some time to get them 'Add product from link' functionality operational.

I'll try remember to message you here in a few weeks once that's live.

Enjoy your holiday :)

**Comment by bwims** [comment]
Hi Alex, I'm back.

This screenshot should make it clear what the problem is!

If I click "Open Sign in"  I get a quick flash of the login prompt and another tab showing my details.  Click Moonsift again, and it says I'm not logged in.

It might be worth you putting an option in place to create a debug/log file.

I appreciate this was written for Chrome rather than Edge, but since both are based on Chromium, that would be a very limiting factor.

Regards

B.

![image](https://github.com/JakeSteam/blog-personal/assets/18516073/ebce9135-cd78-4846-92ec-914e073b5f2c)

**Reply by bwims** [reply]
Also, sorry to say, I now have an acceptable alternative to Amazon assistant, so will be uninstalling the extension.  I wish you well for your future development!

B.

**Reply by AlexMoonsift** [reply]
Great to hear that you found something that works for you and thank you for your warm wishes!

Apologies we weren't fast enough to find a solution for your SRware Iron browser and that our Chrome solution wouldn't work on your version of Edge - we will investigate this more in future.

We've had to just focus on building a great solution for Chrome for now - which is already complex, but Moonsift is actually getting rave reviews from people on Chrome now.

We are still working on enabling people to 'generate items by pasting a link', taking longer than we expected to do it well as we want to release something good.

All the best to you and thanks for the conversation :)

**Comment by plbecker** [comment]
Hey Jake,

Wanted to add one more option here - have you tried [Listful.com](https://www.listful.com/)? I am part of the team working on it. 
We have a website, Chrome extension and a native iOS app (native Android coming soon!) and a few added features such as:
- Automatic price comparisons across retailers for all wishes on your list
- Collaboration + sharing: Add your partner, family or friends as a collaborator and edit your list together
- Personalized gift recommendations based on your list and extensive gift ideas for any occasions (see our [Gift Ideas](https://www.listful.com/gift-ideas))
- In-app browsing + one-click adding (without having to leave the app)
- Internationalization for other 6+ other languages

People use Listful for wishlists (birthday, Christmas etc), shopping / inspiration lists and as a registry (weddings, baby showers).

Would love to get your thoughts! If you are missing any functionality, we'd love to add it.

**Comment by wishrapp** [comment]
Hi Jake, great roundup — it's still one of the first things that comes up when people go looking for wishlist alternatives, which says a lot!

Full disclosure: I'm from the team behind [Wishr.com](https://wishr.com), so consider us the new kid on the block here 😄 We built it because we kept running into the same walls you describe in the post — either too basic, too cluttered, or clearly built around affiliate links.

A few things we do a little differently:

- **Genuinely free and ad-free** — no cost, no adverts, no affiliate-link clutter (we know that was your dealbreaker!)
- **Add anything, share with a simple link** — items from any store, and each list gets its own clean shareable URL
- **Per-occasion privacy** — granular control over who sees what, per list/occasion, so your Secret Santa group doesn't see your wedding registry (and vice versa)
- **Group gifting** — friends and family can chip in together on bigger items
- **Built-in Secret Santa** — draw names and match people to their wishlists in one place
- **Wishr Match** — a little game that helps friends figure out which gift you'd actually love
- **[Mood Board maker](https://wishr.com/mood-board)** — create a visual collage from your wishlist items, for when the "wish" is more of a vibe than a specific product

If you ever revisit this post, we'd be honoured to be put through the same wringer as the others — and honest feedback is very welcome either way. Thanks for the write-up!
